api/index.py

```
@@ -0,0 +1,2 @@
+from backend.main import app
+if __name__ == "__main__": app.run()
```

Ruff (0.14.6)

Multiple statements on one line (colon)

(E701)